coursefunctions.h

```diff
@@ -441,42 +441,63 @@ void TestVertexShader(Buffer2D<PIXEL> & target)
 
         PIXEL colors[3] = {0xffff0000, 0xff00ff00, 0xff0000ff};
         // Your code for 'colorAttributes' goes here
+        colorAttributes[0].var.insert(std::pair<char, double>('r', 1.0));
```

Comment:
This was an interesting solution to adding attributes! I guess this way it doesn't matter which order they go in.